File: Complementares/C/fibonacci.c
/*
 * Autor: Othon Alberto
 * Data: 30/03/2016
 *
 * Exercício complementar passado durante a aula
 *
 * Descrição:
 * Implementar a sequência de Fibonacci */


#include <stdio.h>

int main(){
    
    int nAtual = 0, nAnt = 1, nGer;
    int x, n;
    
    printf("Escreva a quantidade de número que deseja: ");
    scanf("%d", &n);
    
    for (x=0; x<n; x++){
        printf("%i \n", nAtual);
        
        nGer = nAnt + nAtual;
        nAnt = nAtual;
        nAtual = nGer;
    }
}

File: Complementares/JAVA/Fibonacci.java
/*
 * Autor: Matheus Scaketti
 * Data: 30/03/2016
 * 
 * Exercício: Sequência de Fibonacci
 * 
 * Descrição:
 * Exercício passado na aula de algoritmos, mostrando a sequência de fibonacci
 * com o comando for.
*/

import java.util.Scanner;

public class Fibonacci {

	public static void main(String[] args) {
		int nAtual = 0, nAnt = 1, nGer = 0, aux, cont;
		
		Scanner leitor = new Scanner(System.in);
		
		cont = leitor.nextInt();
		leitor.close();
		
		for (aux = 0; aux < cont; aux++){
			System.out.println(nAtual);
			nGer = nAnt + nAtual;
			nAnt = nAtual;
			nAtual = nGer;
		}

	}

}
